pro/src/pages/Home/Offerers/VenueCreationLinks.tsx

```diff
@@ -52,24 +51,12 @@ const VenueCreationLinks = ({ offerer }: VenueCreationLinksProps) => {
           onClick={() => {
             logEvent?.(Events.CLICKED_CREATE_VENUE, {
               from: location.pathname,
-              is_first_venue: !hasPhysicalVenue && !hasVirtualOffers,
+              is_first_venue: false,
```

Comment:
(Question): Pourquoi ce changement ?

Reply:
Si tu regardes un peu au dessus on fait 

```
  if (!hasPhysicalVenue) {
    return
  }
```

Donc cette condition est tout le temps false

Reply:
D'accord, je vois, mais ça ne devrait pas être `!hasVirtualOffers` dans ce cas ?

Reply:
Si ça passe au dela du return ça veut dire que `!hasPhysicalVenue === false` donc forcément `!hasPhysicalVenue && !hasVirtualOffers === false` :wink:

Reply:
Je suis d'accord, mais je pense que le plus safe, serait de changer la ligne en ça
```suggestion
              is_first_venue: !hasVirtualOffers,
```

Reply:
Oui effectivement si la condition au dessus change ou on déplace ce bout de code c'est une bonne idée, c'est corrigé :+1: